transport/grpc/outbound.go

```diff
@@ -229,3 +229,79 @@ func invokeErrorToYARPCError(err error, responseMD metadata.MD) error {
 	}
 	return yarpcerrors.Newf(code, message).WithName(name)
 }
+
+// CallStream implements transport.StreamOutbound#CallStream.
+func (o *Outbound) CallStream(ctx context.Context, requestMeta *transport.RequestMeta) (transport.ClientStream, error) {
+	if err := o.once.WaitUntilRunning(ctx); err != nil {
+		return nil, err
+	}
+	start := time.Now()
```

Comment:
nit: why not just inline `time.Now()` in the `o.stream` call?

Reply:
To maximize the time subtracted when we compute the timeout for outbound calls on the same context. Provides the most real information about how much time the request was in process.

Reply:
Sounds good